src/components/NurseryWithdrawals/PlantingProgressList.tsx

```diff
@@ -92,7 +93,9 @@ export default function PlantingProgressList({
   const [selectedRows, setSelectedRows] = useState<any[]>([]);
   const dispatch = useAppDispatch();
   const [requestId, setRequestId] = useState<string>('');
+  const [selectedZoneIds, setSelectedZoneIds] = useState<number[]>();
   const updatePlantingResult = useAppSelector((state) => selectUpdatePlantingsCompleted(state, requestId));
+  const subzonesStatisticsResult = useAppSelector((state) => selectZonesHaveStatistics(state, selectedZoneIds));
```

Comment:
To see data here, we need to dispatch a call to request observation results in the parent component, example: https://github.com/terraware/terraware-web/blob/main/src/components/Observations/index.tsx#L52

Also may need to refresh that data if planting complete is updated

Reply:
Here when you make the dispatch call, you can use '-1' as the site id (to fetch all results).